packages/imperative/__tests__/OverridesLoader.test.ts

```diff
@@ -57,7 +62,58 @@ describe("OverridesLoader", () => {
             expect(CredentialManagerFactory.initialize).toHaveBeenCalledTimes(0);
         });
 
-        it("should load the default when not passed any configuration and keytar is present in dependencies.", async () => {
+        it("should not set a credential manager for CLI if there are no overrides and keytar is present in dependencies", async () => {
+            const config: IImperativeConfig = {
+                name: "ABCD",
+                overrides: {},
+                productDisplayName: "a fake CLI"
+            };
+
+            // Fake out package.json for the overrides loader
+            const packageJson = {
+                name: "host-package",
+                dependencies: {
+                    keytar: "1.0"
+                }
+            };
+
+            jest.spyOn(AppSettings, "initialized", "get").mockReturnValue(true);
+            jest.spyOn(AppSettings, "instance", "get").mockReturnValue({
+                getNamespace: jest.fn()
+            } as any);
```

Comment:
Will this cause problems to new tests added that depend on the `AppSettings.instance` if this mocked implementation isn't restored after the test ?

Reply:
I believe the `jest.restoreAllMocks` in the afterAll block should handle this, by resetting all `spyOn` mocks including this one to their original implementation.